api/routes/index.js

```diff
@@ -64,24 +70,27 @@ module.exports = function (passport) {
 			User.findOneAndRemove({ 'username': user.username }, function (err, user) {
 				if (err) { return res.send(err); }
 				console.log('Deleting user: ', user.username);
-				return res.send({ message: 'Successfully deleted user' });
+				res.setHeader("Content-Type", "application/json");
+				return res.json({ message: 'Successfully deleted user' });
 			});
 		})(req, res, next);
 	});
 
 	/* Handle Login POST */
 	router.post('/login', function (req, res, next) {
-		console.log('loggin in authentication!');
 		passport.authenticate('login', function (err, user, info) {
-			console.log('loggin in hello');
-			if (err) { return next(err); }
+			if (err) { return res.send(err); }
```

Comment:
Maybe we can get into the habit of sending everything back as json, in this case: `res.json({error: err})`, which would make it simpler to consume on the front-end.